src/status_im2/common/standard_auth/style.cljs

```diff
@@ -0,0 +1,46 @@
+(ns status-im2.common.standard-auth.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
```

Comment:
the formatting is very difficult to read. 
it's much easier if you align the map keys across this pr 👍 
```
(def inner-container
  {:flex 1 
   :padding-horizontal 20})
```

Reply:
@J-Son89 Updated the formatting!